priv/script.sh

```diff
@@ -2,441 +2,490 @@
 
 set -eu
 
+elixir_version=1.17.3-otp-27
+erlang_version=27.1.2
+phoenix_version=1.7.14
+postgres_version=15.1
+
+OS="$(uname -s)"
+case "$OS" in
+Linux*) os_type=Linux ;;
+Darwin*) os_type=macOS ;;
+*)
+  printf >&2 "Unsupported OS: %s\n" "$OS"
+  exit 1
+  ;;
+esac
+
+has() {
```

Comment:
This simplifies reading some tests. I also modified a couple of cases where `mise which` was used to match the versions installed. This can be simplified if you just want to use *any* installed version of Elixir and/or OTP.